test/browsertype-basic.spec.ts

```diff
@@ -23,7 +23,7 @@ it('browserType.executablePath should work', test => {
 }, async ({browserType}) => {
   const executablePath = browserType.executablePath();
   expect(fs.existsSync(executablePath)).toBe(true);
-  expect(fs.realpathSync(executablePath)).toBe(executablePath);
+  expect(fs.realpathSync(executablePath)).toBe(fs.realpathSync(executablePath));
```

Comment:
Hm!

Reply:
let me think about it, will come back tomorrow.